azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -74,14 +77,16 @@ public void testSendEmail() throws Exception {
     Assert.assertNotNull(flow);
 
     final ExecutableFlow exFlow = new ExecutableFlow(this.project, flow);
-    final Emailer emailer = new Emailer(this.props);
+    final CommonMetrics commonMetrics = new CommonMetrics(new MetricsManager(new MetricRegistry()));
```

Comment:
Should CommonMetrics be placed as a private member as it doesn't change? Just a minor comment. Either way works.